### PR TITLE
text/string generators become one + removing to_csv from dataset_generator

### PR DESCRIPTION
**Pull request by lizlouise1335**
- text and string generators are now just one overarching string generator 
- all files affected by this change have been remedied 
- to_csv has been removed from dataset_generator
- testing files have been updated to reflect all above changes

### CODE REVIEW

File: synthetic_data/dataset_generator.py [hunk 1]
```diff
@@ -12,12 +12,11 @@
 from synthetic_data.distinct_generators.datetime_generator import random_datetimes
 from synthetic_data.distinct_generators.float_generator import random_floats
 from synthetic_data.distinct_generators.int_generator import random_integers
-from synthetic_data.distinct_generators.text_generator import random_string, random_text
+from synthetic_data.distinct_generators.text_generator import random_string
```
Comment (taylorfturner) [inline synthetic_data/dataset_generator.py:15]: nice

File: synthetic_data/dataset_generator.py [hunk 2]
```diff
@@ -12,12 +12,11 @@
...
 
 
 def convert_data_to_df(
     np_data: np.array,
-    path: Optional[str] = None,
```
Comment (taylorfturner) [inline synthetic_data/dataset_generator.py:20]: nice

File: tests/test_dataset_generator.py [hunk 3]
```diff
@@ -89,52 +81,21 @@ def test_generate_custom_dataset(self):
...
-                    "010001100011001001101011000010000111011010011000110000111110000000000000101101111011101000011001010111110100000010100000000100110001001000110010010011110001111011101111101001011111000000000011000110100000011010111001000001000110111011111000011111010011011000111100000001111100011000011111000000001000",
-                    "000011011010101010010011011001001111001000000001111110111101010000011101101000000110111001000101110001011100101110001000100001110101001011110110101000110101000100100010011011000010000111101001111000000011011000100011010100001111111111110010011101110010101010010010110011110011001010100000111111110001",
-                    "101111001110001101001010001110100100010010001011110100110000100001000100010100110001110010000111100100010010011010011111000101001110111001111000011011011011111100101010110111100110000111010000100001100111111111010001011100010010100111101010010100011011110110101110111111111000000100110001110011010000",
```
Comment (taylorfturner) [inline tests/test_dataset_generator.py:102]: so beautiful ❤️

File: tests/test_dataset_generator.py [hunk 4]
```diff
@@ -89,52 +81,21 @@ def test_generate_custom_dataset(self):
...
+            dict(zip(["int", "dat", "str", "cat", "flo"], expected_data))
         )
         df = generate_dataset(
             self.rng,
             columns_to_generate=self.columns_to_gen,
             dataset_length=self.dataset_length,
-            path=None,
         )
+        for blah in df:
+            print(list(df[blah]))
```
Comment (taylorfturner) [inline tests/test_dataset_generator.py:100]: yikes ☠️

Comment (taylorfturner) [reply tests/test_dataset_generator.py:100]: 😂

File: synthetic_data/distinct_generators/text_generator.py [hunk 5]
```diff
@@ -10,7 +10,7 @@ def random_string(
     rng: Generator,
     chars: Optional[List[str]] = None,
     str_len_min: int = 1,
-    str_len_max: int = 256,
+    str_len_max: int = 1000,
```
Comment (taylorfturner) [inline synthetic_data/distinct_generators/text_generator.py:13]: I think we should clarify the docstring so its clear the value here is one above the largest per the [docs](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.integers.html#:~:text=If%20provided%2C%20one%20above%20the%20largest%20(signed)%20integer%20to%20be%20drawn%20from%20the%20distribution%20(see%20above%20for%20behavior%20if%20high%3DNone).%20If%20array%2Dlike%2C%20must%20contain%20integer%20values)

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: tests/distinct_generators/test_text_generator.py [hunk 6]
```diff
@@ -3,7 +3,7 @@
 import numpy as np
 import pandas as pd
 
-from synthetic_data.distinct_generators.text_generator import random_string, random_text
+from synthetic_data.distinct_generators.text_generator import random_string
 
 
 class TestTextGeneratorFunctions(unittest.TestCase):
```
Comment (taylorfturner) [inline tests/distinct_generators/test_text_generator.py:9]: rename

File: tests/distinct_generators/test_text_generator.py
Comment (taylorfturner) [inline tests/distinct_generators/test_text_generator.py:1]: rename file

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: synthetic_data/dataset_generator.py [hunk 7]
```diff
@@ -59,16 +52,13 @@ def generate_dataset(
...
 
     :return: pandas DataFrame
     """
     gen_funcs = {
         "integer": random_integers,
         "float": random_floats,
         "categorical": random_categorical,
-        "text": random_text,
         "datetime": random_datetimes,
         "string": random_string,
```
Comment (taylorfturner) [inline synthetic_data/dataset_generator.py:63]: and its `string` here

**Review by taylorfturner — APPROVED** [review]

File: tests/test_dataset_generator.py [hunk 8]
```diff
@@ -89,52 +81,18 @@ def test_generate_custom_dataset(self):
                 ]
             ),
             np.array(
-                [
-                    "010100011010001000001100000001111101111111110110001011101100001010000100101100001010101000100010101010101011000110010110110000011101001011101110011010000011111010001100000000110001100100000000000111001110110110101010110010010000101101110001000101010110101101001011101011110110100110100000111101010101",
```
Comment (JGSweets) [inline tests/test_dataset_generator.py:93]: love this removal

File: tests/test_dataset_generator.py [hunk 9]
```diff
@@ -89,52 +81,18 @@ def test_generate_custom_dataset(self):
...
-                    "000011011010101010010011011001001111001000000001111110111101010000011101101000000110111001000101110001011100101110001000100001110101001011110110101000110101000100100010011011000010000111101001111000000011011000100011010100001111111111110010011101110010101010010010110011110011001010100000111111110001",
-                    "101111001110001101001010001110100100010010001011110100110000100001000100010100110001110010000111100100010010011010011111000101001110111001111000011011011011111100101010110111100110000111010000100001100111111111010001011100010010100111101010010100011011110110101110111111111000000100110001110011010000",
-                ]
-            ),
-            np.array(
-                ["01", "010", "110", "0111", "1001", "001", "0100", "0111", "11", "101"]
+                ["10", "0001", "0100", "10", "000", "100", "00", "01", "1110", "1111"]
```
Comment (JGSweets) [inline tests/test_dataset_generator.py:84]: changing bc string isn't as long

File: tests/distinct_generators/test_text_generator.py [hunk 10]
```diff
@@ -1,49 +0,0 @@
-import unittest
```
Comment (JGSweets) [inline tests/distinct_generators/test_text_generator.py:1]: would suggest doing text over string bc it matches DP

**Review by JGSweets — APPROVED** [review]

**Pull request merged**